lang/ghc/Portfile

```diff
@@ -64,6 +64,7 @@ if {[variant_isset "bootstrap"]} {
                     port:autoconf \
                     port:automake \
                     port:python${python3_version_nickname} \
+                    port:py${python3_version_nickname}-sphinx \
```

Comment:
this only adds this for the 'bootstrap' variant. is this intentional ?

Reply:
Yes. The "nobootsrap" doesn't build the dox, so it's only necessary when building from source.